### ISSUE TITLE
Hmr on windows: File not served

### ISSUE THREAD

**Issue by kpilard** [JeffreyWay/laravel-mix#78]
Hey

When i call `npm run hmr`on my Mac, I can open the js file with hmt on `localhost:8080/js/app.js`.
On both of my windows computers I get `Cannot GET /js/bundle.js`

I think the problem is here https://github.com/JeffreyWay/laravel-mix/blob/master/src/Mix.js#L96

Return of the Object on Window: 
```
{ 
  path: '/',
  filename: '\\js\\[name].js',
  publicPath: 'http://localhost:8080/' 
}
```

on my Mac it is:
```
{ 
  path: '/',
  filename: '/js/[name].js',
  publicPath: 'http://localhost:8080/' 
}
```

Edit: It's working hardcoded on windows
```
return {
            path: this.hmr ? '/' : this.publicPath,
            filename: '/js/[name].js',
            publicPath: this.hmr ? 'http://localhost:8080/' : './'
        }
```

**Comment by bvangraafeiland** [comment]
I've fixed this in #97, which replaces all uses of backslashes to forward slashes.

**Comment by kpilard** [comment]
it works fine on my compute with v0.51r :) thank you
